sys/crypto/modes/ccm.c

```diff
@@ -167,8 +175,19 @@ static inline int _fits_in_nbytes(size_t value, uint8_t num_bytes)
     return (value >> shift) <= 1;
 }
 
+/* Valid mac_length are 4, 6, 8 ... 16 octets */
+static int _validate_mac_length_ccm(uint8_t mac_length)
+{
+    return (mac_length % 2 != 0 || mac_length < 4 || mac_length > 16);
```

Comment:
Returns `0` if valid and `1` if invalid, right? Feels a bit counter intuitive, maybe add a comment?

Reply:
Fixed by returning the intuitive value.